MockServer/src/main/java/com/github/dedis/student20_pop/mockserver/WebSocketServer.java

```diff
@@ -0,0 +1,67 @@
+package com.github.dedis.student20_pop.mockserver;
+
+import org.glassfish.tyrus.server.Server;
+
+import java.io.IOException;
+import java.util.Locale;
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import javax.websocket.CloseReason;
+import javax.websocket.DeploymentException;
+import javax.websocket.OnClose;
+import javax.websocket.OnMessage;
+import javax.websocket.OnOpen;
+import javax.websocket.Session;
+import javax.websocket.server.ServerEndpoint;
+
+import static java.lang.String.format;
+
+public class WebSocketServer {
+
+    public static void main(String[] args) {
+
+        Server server = new Server("localhost", 2020, "", PoPServerEndpoint.class);
```

Comment:
I think it's better if we don't hardcode the string values, and instead create final static variables for them. I would suggest the same for line 29, 48, 54 and 64

Reply:
Done !